pkg/awsutils/awsutils.go

```diff
@@ -223,8 +224,31 @@ func prometheusRegister() {
 	}
 }
 
+type AWSStrings struct {
+
+}
+//StringSet is a set of strings
+type StringSet struct {
+	sync.RWMutex
+	data 	sets.String
+	// dataNew var is only used to check difference
+	dataNew sets.String
```

Comment:
this field isn't needed.

the StringSet should be comparable to another StringSet object instead of having a StringSet store an "old" and "new" set of strings.